src/Run/Setup.cs

```diff
@@ -271,14 +268,14 @@ private string GetTool(Command commandToExecute, string os, string configPath, L
             if (Tools.ContainsKey(toolname))
```

Comment:
Instead of doing a contains here and then various indexing below. I would do a Tools.TryGetValue here and just use the Tool it finds below.